.cirrus.yml

```diff
@@ -420,15 +448,15 @@ task:
 
 task:
   matrix:
-    - name: test-windows-mingw64
+    - name: 'Testing VM Image: windows-ci-mingw64'
       depends_on:
         - windows-ci-mingw64
       compute_engine_instance:
         image_project: $GCP_PROJECT
         image: family/pg-ci-windows-ci-mingw64
         platform: windows
 
-    - name: test-windows-mingw64-amd64-container
+    - name: 'Testing Docker Image: windows_ci_mingw64'
```

Comment:
I think s/Docker Image/Container Image/ would be good, here and in earlier locations.